### ISSUE TITLE
Allow user to define more than one preset

### ISSUE THREAD

**Issue by DenoBeno** [clarity-h2020/csis#108]
Related to: https://github.com/clarity-h2020/csis/issues/101

Users can choose ONE combination of parameters in the study now. Eventually we should allow them to choose several such presets and hide everything that doesn't fit to their presets from the study. E.g., also hide the zillions of the hazard resources that are of no interest for the user  and only show those that are.

(on hold for now)

**Comment by patrickkaleta** [comment]
One of the first steps here would be to modify the `$studyInfo` object to be able to store multiple study presets. This is very easy to do and I tested it already locally. The `studyInfo `object would then look like this:
![multiple-presets](https://user-images.githubusercontent.com/42167036/66473376-6ce9df00-ea8f-11e9-95fb-60a3d65c8a91.png)

@p-a-s-c-a-l  and @therter: How much work would it mean for you two if we turned the `study_presets `attribute into an array, but left the maximum number of presets in the Drupal settings at 1 (for now)?

**Comment by p-a-s-c-a-l** [comment]
not so much. I would temporarily keep the old and the new structure in parallel to allow a smooth transition.

Perhaps we should rename 'preset' to 'scenario' (not to be confused with emission scenarios) because it's essentially scenario comparison what we want to do. Especially comparison of base scenarios  (= those without adaptation options applied) vs adaptation scenarios (atm called 'study_variants').

Maybe we have also to rethink how we handle 'study_variants' and what we what to achieve: Allow the user to compare across  base scenarios (rcp26 vs rcp85) or just across base and adaptation scenarios. The later one is actually needed, the first one maybe not.

**Comment by DenoBeno** [comment]
Preset => scenario? I have no issue with this change.

I don't know what users may want to see. Some could be interested to see how the future differs in the case that we do nothing and in the case that we do something against the climate change. If they can choose the "presets" or "scenarios" in any way, it's not my problem either. Maybe it would be possible to pre-define some scenarios that we consider useful and then let the users change this if they want.

**Comment by DenoBeno** [comment]
> not so much. I would temporarily keep the old and the new structure in parallel to allow a smooth transition.

If this isn't a big deal, then let us allow the use of several presets (study scenarios?) immediately and see that we support it on more and more places over time.

**Comment by p-a-s-c-a-l** [comment]
>  let us allow the use of several presets (study scenarios?) immediately and see that we support it on more and more places over time.

**that**, however, **is**  a big deal in terms of usability: The user should not be able to chose options which have no effect or have inconsistent effects. 

Btw, we should now invest in stability and relatability of the system so that we can make it public asap, if it's not "perfect" or "feature complete" (it will never be). New features can be added later when we get feedback from **real** end users.

**Comment by DenoBeno** [comment]
OK

**Comment by p-a-s-c-a-l** [comment]
What we would need is a way to define several Study Scenarios and a possibility to select (ATM) *one* of them in map and table display.

ATM we can just *edit* one study scenario in table an map display: 

![grafik](https://user-images.githubusercontent.com/1788226/70704231-e42b3180-1cd1-11ea-99db-e125797ef48c.png)

This should be changed to a drop-down box where we can *select* one of the predefined study scenarios. Editing and creating of study scenarios should only be possible in study context.

**Comment by p-a-s-c-a-l** [comment]
done
